code/modules/mob/living/silicon/ai/death.dm

```diff
@@ -39,7 +39,7 @@
 		callshuttle = 0
 
 	if(callshuttle == 3) //if all three conditions are met
-		emergency_shuttle.call_evac()
+		shuttle_master.emergency.request(null, 0.3, null, "All communication consoles, boards, and AI's have been destroyed.")

```

Comment:
May want ot look at how TG handles this; I dunno if they check for the comm boards, or just the comm's consoles and AI?